magma/uniquification.py

```diff
@@ -23,13 +23,22 @@ class _HashStruct:
     defn_repr: str
     is_verilog: bool
     verilog_str: bool
+    inline_verilog: tuple
 
 
 def _make_hash_struct(definition):
     repr_ = repr(definition)
+    inline_verilog = ((), (), (), ())
+    for s, args, st, prefix in definition._context_._inline_verilog:
+        inline_verilog += (
+            (s, ),
+            (", ".join(str(arg) for arg in args), ),
+            (str(st), ),
+            (prefix, )
+        )
```

Comment:
```suggestion
        inline_verilog += (
            (s,),
            (", ".join(str(arg) for arg in args),),
            (str(st),),
            (prefix,)
        )
```

see https://www.python.org/dev/peps/pep-0008/#pet-peeves